docs/correlated_observation_scaling.rst

```diff
@@ -0,0 +1,133 @@
+Correlated Observations Scaling
+===============================
+
+Configuration
```

Comment:
Configsuite includes `rst` generation functionality based on the content in the schema and also has a sphinx extension that would generate an html page with a slightly more interactive setup.

I would want to point to the documenation page, but it seems even the `latest` does not include the last commit (which is the one containing the documentation).

In particular though, the extension is a bit rough (or the product of the extension is rough). But I think it would be beneficial spending time improving that part instead. We will need it for later (and several jobs in spinningjenny will greatly benefit from an improved extension)